include/deal.II/fe/fe_tools.h

```diff
@@ -1135,7 +1135,7 @@ namespace FETools
    * function, use the add_fe_name() function.  This function does not work if
    * one wants to get a codimension 1 finite element.
    */
-  template <int dim, int spacedim>
+  template <int dim, int spacedim=dim>
```

Comment:
I am fine with your workaround. Sorry, for all these inconveniences.
Just to be sure: What you wanted to say is `FETools::get_fe_by_name<dim, spacedim>` is in 8.4.1, but `FETools::get_fe_by_name<dim>` is not.